flink-ml-iteration/src/main/java/org/apache/flink/iteration/operator/allround/OneInputAllRoundWrapperOperator.java

```diff
@@ -87,7 +90,9 @@ public void setKeyContextElement(StreamRecord<IterationRecord<IN>> record) throw
     @Override
     public void endInput() throws Exception {
         if (wrappedOperator instanceof BoundedOneInput) {
+            setIterationContextRound(iterationContextRound);
```

Comment:
Would it be simpler to do `setIterationContextRound(Integer.MAX_VALUE)`?

Reply:
Yes, you are right. It's safe and simpler. I've updated and please have another look.